waves_gateway/service/coin_transaction_consumer_impl.py

```diff
@@ -178,6 +207,66 @@ def _handle_transaction_receiver(self,
             self._attempt_list_storage.safely_save_attempt_list(attempt_list)
             self._logger.info('Created new attempt list %s', str(attempt_list.attempt_list_id))
 
+    def _transfer_back(self, transaction, receiver: TransactionReceiver, sender: TransactionSender, index,
```

Comment:
In my opinion, the whole _transfer_back_block here is unnecessary code. It is only called if an internal error occurs which can only happen if there is a bug in the framework.  Adding this, may only open new bugs and security holes. @jansenmarc Please take a look. I need a second opinion.

Reply:
Bugs can always arise, so I think it's important to consider something like that to support this cases in favor of the customer.